index.bs

```diff
@@ -12,23 +12,55 @@ Abstract: This API defines an API surface for controlling how audio is rendered
 Markup Shorthands: css no, markdown yes
 </pre>
 
+<pre class=link-defaults>
+spec:html; type:dfn; for:/; text:browsing context
+</pre>
+
 # Introduction # {#introduction}
 
-People consume a lot of media (audio/video) and the Web is one of the primary means of consuming this type of content.
-However, media on the web does not integrate well with the platform.
-The Audio Session API helps to close the gap with platforms that have audio session/audio focus such as Android and iOS.
-This API will help by improving the audio-mixing of websites with native apps, so they can play on top of each other, or play exclusively.
+People increasingly consume media (audio/video) through the Web,
+which has become a primary channel for accessing this type of content.
+However, media on the Web often lacks seamless integration with underlying platforms.
+The Audio Session API addresses this gap by enhancing media handling
+across platforms that support audio session management or similar audio focus features.
+This API improves how web-based audio interacts with other apps,
+allowing for better audio mixing or exclusive playback, depending on the context,
+to provide a more consistent and integrated media experience across devices.
 
-Additionally, on some platforms the user agent will automatically manage the audio session for the site
-based on whether media elements are playing or not and which APIs are used for playing audio.
-In some cases this may not match user expectations, this API provides overrides to authors.
+Additionally, some platforms automatically manage a site's audio session
+based on media playback and the APIs used to play audio.
+However, this behavior might not always align with user expectations.
+This API allows developers to override the default behavior and gain more control over an audio session.
 
-# The {{AudioSession}} interface # {#audiosession-interface}
+# Concepts # {#concepts}
+
+An <dfn>audio session</dfn> represents the playback of audible media.
+An [=audio session=] can be of a particular [=audio session/type=], and be in a particular [=audio session/state=].
+An [=audio session=] manages the audio for a set of sources and sinks, named audio session <dfn data-lt="element" for="audio session">elements</dfn>.
+
+An audio session [=audio session/element=] has a number of properties:
+
+* A <dfn data-lt="default type" for="audio session">default type</dfn>, which is used to compute the [=audio session=] [=audio session/type=], in case of "{{AudioSessionType/auto}}".
+* An <dfn>audible flag</dfn>, which is either `true` if the element is playing/recording audio or `false` otherwise.
 
-An <dfn>audio session</dfn> represents the playback of auditory media.
-An audio session can be of particular [=audio session/type=] and in a given [=audio session/state=].
+An audio session [=audio session/element=] is an <dfn>audible element</dfn> if its [=audible flag=] is `true`.
```

Comment:
This is going to be a bit confusing because "element" and "audible element"... maybe: "`<dfn>element that is audible</dfn>`" ... just for clarity?

Reply:
I'd like to keep it for this PR this way since audible element is used in other commits.
Once other commits land, how about reducing to <dfn>audible</dfn>?

Reply:
Reducing to only `audible` sounds good to me.

Reply:
I filed https://github.com/w3c/audio-session/issues/26